packages/compass-indexes/src/components/create-index-form/create-index-form.tsx

```diff
@@ -70,135 +74,134 @@ export type CreateIndexProps = {
+
+  toggleIsUnique,
+  toggleIsSparse,
+
+  toggleUseIndexName,
+  toggleUseTtl,
+  toggleUsePartialFilterExpression,
+  toggleUseWildcardProjection,
+  toggleUseCustomCollation,
+  toggleUseColumnstoreProjection,
+
+  nameChanged,
+  ttlChanged,
+  partialFilterExpressionChanged,
+  wildcardProjectionChanged,
+  collationStringChanged,
+  columnstoreProjectionChanged,
+
+  updateFieldName,
+  updateFieldType,
+  addField,
+  removeField,
+  createNewIndexField,
+}: CreateIndexFormProps) {
+  return (
+    <>
       <div
-        data-testid="create-index-modal-options"
-        className={createIndexModalOptionStyles}
+        className={createIndexModalFieldsStyles}
+        data-testid="create-index-form"
       >
-        <UniqueIndexCheckbox
-          isUnique={this.props.isUnique}
-          toggleIsUnique={this.props.toggleIsUnique}
-        />
-        <IndexNameCollapsibleFieldSet
-          useIndexName={this.props.useIndexName}
-          toggleUseIndexName={this.props.toggleUseIndexName}
-          indexName={this.props.name}
-          nameChanged={this.props.nameChanged}
-        />
-        <TTLCollapsibleFieldSet
-          useTtl={this.props.useTtl}
-          toggleUseTtl={this.props.toggleUseTtl}
-          ttl={this.props.ttl}
-          ttlChanged={this.props.ttlChanged}
-        />
-        <PartialFilterCollapsibleFieldSet
-          usePartialFilterExpression={this.props.usePartialFilterExpression}
-          toggleUsePartialFilterExpression={
-            this.props.toggleUsePartialFilterExpression
-          }
-          partialFilterExpression={this.props.partialFilterExpression}
-          partialFilterExpressionChanged={
-            this.props.partialFilterExpressionChanged
-          }
-        />
-        <WildcardProjectionCollapsibleFieldSet
-          useWildcardProjection={this.props.useWildcardProjection}
-          toggleUseWildcardProjection={this.props.toggleUseWildcardProjection}
-          wildcardProjection={this.props.wildcardProjection}
-          wildcardProjectionChanged={this.props.wildcardProjectionChanged}
-        />
-        <CustomCollationCollapsibleFieldSet
-          useCustomCollation={this.props.useCustomCollation}
-          toggleUseCustomCollation={this.props.toggleUseCustomCollation}
-          collationString={this.props.collationString}
-          collationStringChanged={this.props.collationStringChanged}
-        />
-        {hasColumnstoreIndexesSupport(this.props.serverVersion) && (
-          <ColumnstoreProjectionCollapsibleFieldSet
-            useColumnstoreProjection={this.props.useColumnstoreProjection}
-            toggleUseColumnstoreProjection={
-              this.props.toggleUseColumnstoreProjection
-            }
-            columnstoreProjection={this.props.columnstoreProjection}
-            columnstoreProjectionChanged={
-              this.props.columnstoreProjectionChanged
-            }
+        <Body weight="medium" className={indexFieldsHeaderStyles}>
```

Comment:
Shouldn't this be a label?

For the other fields we do this: `<Label htmlFor={labelId}>Create unique index</Label>`

Reply:
Oh, I only meant that style-wise it would have been nice to use `<Label>` instead of trying to reproduce one by using a `Body` with the same weight and margin of a label, I think is fine for now, just something that shouldn't be necessary.